packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -1707,6 +1707,23 @@ export class ExpressApp {
       });
     });
 
+    router.post('/v1/service/changelly/getTransactions', (req, res) => {
+      let server;
+      try {
+        server = getServer(req, res);
+      } catch (ex) {
+        return returnError(ex, res, req);
+      }
+      server
+        .changellyGetTransactions(req)
+        .then(response => {
+          res.json(response);
+        })
+        .catch(err => {
+          if (err) return returnError(err, res, req);
```

Comment:
Looks like this is a copy n paste from elsewhere, but when would `!err` be true? And if it were, we'd just never respond to the request? 

I think we should get rid of the `if (err)` condition and just `return returnError(...);` .... maybe pass in `err || 'unknown'` if we're concerned about `!err` being true.

Reply:
Done. Ty!
I only changed the response for this particular request. What do you think if I make the changes for all the pending requests later in a separate PR?